src/common/src/hash/dispatcher.rs

```diff
@@ -12,10 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use num256::Int256;
```

Comment:
nit: this looks like an external crate, but is it actually the mod imported on line 20 below ...